virttest/qemu_devices/qdevices.py

```diff
@@ -1580,17 +1582,14 @@ def format_params(self, params):
         :param params: chardev test params.
         :return dict: formated params only include suppprt options.
         """
-        for opt in ["server", "telnet", "nowait",
-                    "ipv4", "ipv6", "nodelay"]:
-            if params.get(opt) in ["yes", "on", True]:
-                params[opt] = "NO_EQUAL_STRING"
-            elif opt in params:
-                del params[opt]
-        for opt in ["mux", "signal"]:
-            if params.get(opt) in ["yes", "on", True]:
-                params[opt] = True
-            elif params.get(opt) in ["no", "off", False]:
-                params[opt] = False
+        # TODO: The 'delay' option for socket character devices is deprecated
+        #  from qemu 6.0, so might be removed from here later
+        for opt in ["server", "telnet", "wait",
+                    "ipv4", "ipv6", "nodelay", "mux"]:
```

Comment:
Missed `signal` option

Reply:
FYI
`-chardev stdio,id=id[,mux=on|off][,signal=on|off][,logfile=PATH][,logappend=on|off]`

Reply:
@yanan-fu Actually there are many other boolean options for the other backend, like, you see `logappend=on|off`, or `[,abstract=on|off][,tight=on|off]` for `-chardev socket`, I was thinking that not necessarily include them all, user can directly define them to `on`, `off` instead of `yes`, `no`, i just keep those ones that I think likely to be used and properly might be set to `yes`, `no`. What do you think?

Reply:
Yes, I do not means to support all of those options, but keep the old options.  `signal` was removed in this patch, so i comment this option.

Reply:
@yanan-fu Actually there are many other boolean options for the other backend, like, you see `logappend=on|off`, or `[,abstract=on|off][,tight=on|off]` for `-chardev socket`, I was thinking that not necessarily include them all, user can directly define them to `on`, `off` instead of `yes`, `no`, i just keep those ones that I think likely to be used and properly might be set to `yes`, `no`. What do you think?

Reply:
@yanan-fu Add `signal` back, please help re-review, thanks.